src/Place/Pages/PPA002Place/Place.tsx

```diff
@@ -0,0 +1,128 @@
+import { PCT012MPlaceOne } from '@/Place/Models/PDB01MPlace/Contexts/PCT012MPlaceOne/MPlaceOne';
+
+export const PPA002Place = ({ route, navigation }: NativeStackScreenProps<BottomTabParamList, 'PPA002Place'>) => {
+	const { place, setPlaceID, isLoading } = useContext(PCT012MPlaceOne);
+	const { place_id, language } = route.params;
+	const { onCreateItineraryClicked } = PPA002PlaceController(route.params);
+
+	useEffect(() => {
+		setPlaceID(place_id);
+	}, [place_id, setPlaceID]);
+
+	// TODO: https://github.com/Ayato-kosaka/spelieve/issues/321 存在しないplace_idはずっとローディング
+	if (isLoading) {
+		return <ActivityIndicator animating />;
+	}
+	if (!place) {
+		return <Text style={styles.infoText}>{i18n.t('Place Not Found')}</Text>;
+	}
+
+	return (
+		<ScrollView>
+			<View style={styles.container}>
+				<View>
+					<PMC01201GoogleMapPlaceOne />
+					<PMC01203PlaceImage />
+					<Text style={styles.infoText}>{place.name}</Text>
+					<BookIcon name="open-book" size={50} onPress={() => onCreateItineraryClicked(place.name)} />
+					<FetherIcon name="map-pin" size={20}>
+						<Text style={styles.infoText}>{place.formatted_address}</Text>
+					</FetherIcon>
+					<WebIcon name="web" size={20}>
+						<Text
+							style={styles.urlLink}
+							onPress={() => {
+								// eslint-disable-next-line
+								Linking.openURL(`${place.website || ''}`);
+							}}>
+							{place.website ? place.website : i18n.t('No Web Infomation')}
+						</Text>
+					</WebIcon>
+					<FetherIcon name="phone-call" size={20}>
+						<Text style={styles.infoText}>
+							{place.formatted_phone_number ? place.formatted_phone_number : i18n.t('No Tel Information')}
+						</Text>
+					</FetherIcon>
+					<View>
+						<TimeIcon name="time-outline" size={20}>
+							<Text style={styles.infoText}>{i18n.t('Opening Hours')}</Text>
+						</TimeIcon>
+						{Array.isArray(place.openingHours) ? (
+							<FlatList
+								data={place.openingHours}
+								renderItem={(itemData) => {
+									const [day, time] = itemData.item;
+									return (
+										<View>
+											<Text style={styles.infoText}>
+												{day} {time}
+											</Text>
+										</View>
+									);
+								}}
+								numColumns={1}
+							/>
+						) : (
+							<Text style={styles.infoText}>{place.openingHours}</Text>
+						)}
+						<Text
+							style={styles.urlLink}
+							onPress={() => {
+								// eslint-disable-next-line
+								Linking.openURL(`${place.mapUrl}`);
+							}}>
+							{i18n.t('show more')}
+						</Text>
+					</View>
+				</View>
+				<View>
+					<Text style={styles.infoText}>{i18n.t('Customer Reviews')}</Text>
+					<Rating type="star" readonly jumpValue={0.1} startingValue={place.rating} />
+					<Text
+						style={styles.urlLink}
+						onPress={() => {
+							// eslint-disable-next-line
+							Linking.openURL(`${place.mapUrl}`);
+						}}
+					/>
```

Comment:
ここは、「see more info at Google Map」にして〜
```suggestion
					<Text
						style={styles.urlLink}
						onPress={() => {
							// eslint-disable-next-line
							Linking.openURL(`${place.mapUrl}`);
						}}
					>
					{i18n.t('see more info at Google Map')}
					</Text>
```

Reply:
[0d1ac72](https://github.com/Ayato-kosaka/spelieve/pull/326/commits/0d1ac720beaa772a689ec621fe0386c5928cf4d0)